modules/core/04-channel/keeper/handshake.go

```diff
@@ -204,7 +204,7 @@ func (k Keeper) WriteOpenTryChannel(
 
 	k.SetChannel(ctx, portID, channelID, channel)
 
-	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", "NONE", "new-state", "TRYOPEN")
+	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", types.UNINITIALIZED.String(), "new-state", types.TRYOPEN.String())
```

Comment:
In the case of crossing hellos, it may be possible that previous sate could be `UNINITIALIZED` or `INIT`.
But I think support for crossing hellos in opening handshakes was removed at some point 🤔

Reply:
So I should get the previous state like ``k.GetChannel(ctx, portID, channelID).State`` before set new state for the channel ?